uint32 height
uint32 width

uint32 step # widthstep in bytes

Corner[] corners
float32[] cornerDepth

uint8[] data
